integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/AnsibleSteps.java

```diff
@@ -0,0 +1,51 @@
+package com.redhat.service.smartevents.integration.tests.steps;
+
+import java.time.Duration;
+
+import org.assertj.core.api.Assertions;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.Assumptions;
+
+import com.redhat.service.smartevents.integration.tests.context.TestContext;
+import com.redhat.service.smartevents.integration.tests.context.resolver.ContextResolver;
+import com.redhat.service.smartevents.integration.tests.resources.AnsibleResource;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+
+public class AnsibleSteps {
+
+    private TestContext context;
+
+    public AnsibleSteps(TestContext context) {
+        this.context = context;
+    }
+
+    @Given("^Ansible instance url was set$")
+    public void checkAnsibleUrlWasSet() {
+        final String ansibleEndpoint = System.getProperty("ansible.endpoint");
+        Assumptions.assumeTrue(ansibleEndpoint != null);
+        Assumptions.assumeTrue(ansibleEndpoint.length() > 0);
+    }
+
+    @And("^Job template (.*) exists$")
+    public void checkAnsibleJobTemplateExists(String jobTemplateName) {
+        Assertions.assertThat(AnsibleResource.jobTemplateExists(jobTemplateName)).isTrue();
```

Comment:
Can you retrieve a list of job templates and check that the template is among them?
That should give us more context info in case of a typo or similar.